gems/smithy/lib/smithy/views/client/client.rb

```diff
@@ -25,20 +25,23 @@ def gem_version
         end
 
         def require_plugins
-          @plugins.select(&:requirable?).map(&:require_path)
+          @plugins.map do |plugin|
+            "require#{'_relative' if plugin.require_relative?} '#{plugin.require_path}'"
+          end
         end
 
         def add_plugins
-          @plugins.reject(&:default?).map(&:class_name)
+          @plugins.map(&:class_name)
         end
 
         def docstrings
-          docstrings = []
-          docstrings << '@param [Hash] options'
-          # TODO: ensure correct handling of duplicate option definitions
-          @plugins.each do |plugin|
-            docstrings.concat(plugin.docstrings)
+          options = @plugins.map(&:options).flatten.sort_by(&:name)
```

Comment:
How does this handle the order of multiple plugins adding the same option?

Reply:
I believe it is ordered by weld and then by plugin. I verified this by adding a duplicate option and it was able to override. You should also do a verification.